api/controllers/user.js

```diff
@@ -97,6 +130,7 @@ exports.protectedPut = async function (args, res) {
   var User = require('mongoose').model('User');
 
   var user = {
+    idir_user_guid: obj.idir_user_guid ? obj.idir_user_guid : '',
```

Comment:
```suggestion
    idir_user_guid ?? '',
```
is this not the same